### PR TITLE
refactor(text-inputs): update styling of text inputs - INNO-1593

### PR DESCRIPTION
**Pull request by yhuard**
Specs:
- https://webgate.ec.europa.eu/CITnet/confluence/pages/viewpage.action?pageId=885107000
- https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Width+system+for+inputs

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 8cc9868a6b1e99e622a577bbae3ba3553dce8161

https://deploy-preview-1183--europa-component-library.netlify.com

File: src/systems/ec/implementations/vanilla/packages/ec-component-form-text-area/ec-component-form-text-area.scss [hunk 1]
```diff
@@ -25,27 +23,56 @@
...
+    }
+
+    &:active,
     &:focus {
       outline: $border-width-focus solid $border-color-focus;
       outline-offset: -3px;
     }
   }
 
+  .ecl-text-area--md {
```
Comment (emeryro) [inline src/systems/ec/implementations/vanilla/packages/ec-component-form-text-area/ec-component-form-text-area.scss]: we should use the same naming convention for classes as in other components.
so `ecl-text-area--s`, `ecl-text-area--m`, `ecl-text-area--l`

File: src/website/src/pages/ec/components/forms/text-field/docs/code.mdx [hunk 2]
```diff
@@ -53,9 +51,43 @@ import demoContentDefault from '@ecl/ec-specs-text-input/demo/data--default';
...
     label={demoContentDefault.label}
-    placeholder={demoContentDefault.placeholder}
-    helperText="Help message"
+    helperText="This is the input's helper text."
     invalid
-    invalidText="Error message"
+    invalidText="This is the error message"
+  />
+</Playground>
+
```
Comment (emeryro) [inline src/website/src/pages/ec/components/forms/text-field/docs/code.mdx:62]: We should add a paragraph about field sizes on this page

File: src/systems/ec/implementations/vanilla/packages/ec-component-form-text-area/ec-component-form-text-area.scss [hunk 3]
```diff
@@ -25,27 +23,56 @@
     display: block;
     font: $ecl-font-m;
     margin: 0;
-    max-width: $max-width;
-    min-width: $min-width;
+    max-width: 100%;
     padding: calc(#{$padding-vertical} - #{$border-width})
       calc(#{$padding-horizontal} - #{$border-width});
     width: 100%;
```
Comment (emeryro) [inline src/systems/ec/implementations/vanilla/packages/ec-component-form-text-area/ec-component-form-text-area.scss]: I guess there should not be a forced width here

**Review by emeryro — CHANGES_REQUESTED** [review]
firefox adds an extra box-shadow for required textarea, we should remove it
image

**Review by emeryro — APPROVED** [review]

**Pull request merged**